tests/syntactic/equivalent_types_in_negation/equivalent_types_in_negation.dl

```diff
@@ -6,7 +6,6 @@ list_decl(["D", ["E", ["F", nil]]]).
 .type Alias = List
 
 .decl flatten_list(l:List)
-.output flatten_list
```

Comment:
It would be nice to have some outputs in this test. Why would that make the test fail?

Reply:
Yes, probably because I missed something in the test declaration. It waits for zero output where it had two. I probably need to put the number of outputs somewhere.